packages/common/src/TitleBar.ts

```diff
@@ -148,6 +191,18 @@ export class IconBar extends HTMLWidget {
                     return prev;
                 }
             }
+            if ((item as any).fixedHeight) {
+                (item as any).fixedHeight(this.fixedHeight() - (this.buttonMargin() * 2));
+            }
+            if ((item as any).fontSizeStyle) {
+                (item as any).fontSizeStyle(this.buttonFontSizeStyle());
+            }
+            if ((item as any).lineHeightStyle) {
+                (item as any).lineHeightStyle(this.buttonLineHeightStyle());
+            }
+            if ((item as any).margin) {
+                (item as any).margin(this.buttonMargin());
+            }
```

Comment:
These should be inside a: 
```
else if (item instanceof Button) {
...
}
```
Then you won't need to test for each one individually...